### PR TITLE
Spelling: occured -> occurred

### PR DESCRIPTION
**Pull request by perjahn**

### CODE REVIEW

**Comment by elasticcla** [comment]
Hi @perjahn, we have found your signature in our records, but it seems like you have signed with a different e-mail than the one used in your Git [commit](https://github.com/elastic/beats/pull/13743.patch). Can you please add both of these e-mails into your Github profile (they can be hidden), so we can match your e-mails to your Github profile?

**Comment by perjahn** [comment]
Please decline this PR, I will redo the PR with my correct email address.

File: metricbeat/docs/fields.asciidoc [hunk 1]
```diff
@@ -23687,7 +23687,7 @@ type: long
 [float]
 === parse
 
-Parses statistic information that occured in the current session
+Parses statistic information that occurred in the current session
```
Comment (fearful-symmetry) [inline metricbeat/docs/fields.asciidoc]: This is an autogenerated file, it shouldn't be manually edited. It looks like you already edited the source file in the oracle module. Just run `make update` to update this file.

Comment (fearful-symmetry) [reply metricbeat/docs/fields.asciidoc]: The travis failure seems to suggest that _something_ went wrong with the docs. Just run `make update fmt` and try again.

File: metricbeat/vendor/github.com/prometheus/common/expfmt/decode.go [hunk 2]
```diff
@@ -166,7 +166,7 @@ func (sd *SampleDecoder) Decode(s *model.Vector) error {
 // ExtractSamples builds a slice of samples from the provided metric
 // families. If an error occurs during sample extraction, it continues to
 // extract from the remaining metric families. The returned error is the last
-// error that has occured.
+// error that has occurred.
```
Comment (kaiyan-sheng) [inline metricbeat/vendor/github.com/prometheus/common/expfmt/decode.go]: This is under vendor so probably shouldn't be changed.

**Pull request closed without merging**